Mathlib/MeasureTheory/Measure/OuterMeasure.lean

```diff
@@ -368,23 +383,25 @@ theorem univ_eq_zero_iff (m : OuterMeasure α) : m univ = 0 ↔ m = 0 :=
 
 section Supremum
 
-instance : SupSet (OuterMeasure α) :=
+instance instSupSet : SupSet (OuterMeasure α) :=
   ⟨fun ms =>
     { measureOf := fun s => ⨆ m ∈ ms, (m : OuterMeasure α) s
-      Empty := nonpos_iff_eq_zero.1 <| supᵢ₂_le fun m h => le_of_eq m.Empty
-      mono := fun s₁ s₂ hs => supᵢ₂_mono fun m hm => m.mono hs
+      empty := nonpos_iff_eq_zero.1 <| supᵢ₂_le fun m _ => le_of_eq m.empty
+      mono := fun {s₁ s₂} hs => supᵢ₂_mono fun m _ => m.mono hs
       unionᵢ_nat := fun f =>
         supᵢ₂_le fun m hm =>
           calc
             m (⋃ i, f i) ≤ ∑' i : ℕ, m (f i) := m.unionᵢ_nat _
             _ ≤ ∑' i, ⨆ m ∈ ms, (m : OuterMeasure α) (f i) :=
-              ENNReal.tsum_le_tsum fun i => le_supᵢ₂ m hm
+               ENNReal.tsum_le_tsum fun i => by apply le_supᵢ₂ m hm
```

Comment:
Is the indentation change intentional?

Reply:
Without by apply:
```
type mismatch
  le_supᵢ₂ m hm
has type
  ?m.67029 m hm ≤ ⨆ i, ⨆ j, ?m.67029 i j : Prop
but is expected to have type
  ↑m (f i) ≤ ⨆ m, ⨆ h, ↑m (f i) : Prop
```

Reply:
@Ruben-VandeVelde was asking about the space you added immediately before the `ENNReal.tsum_le_tsum`.